src/components/Spinner.tsx

```diff
@@ -2,11 +2,11 @@ import React from 'react';
 import loadingIndicator from '../images/loading-indicator.svg';
 
 //spinner constant to be used everywhere
-export const Spinner = (
-  <div style={{ textAlign: 'center', height: '44px', width: '180px' }}>
-    <img src={loadingIndicator} alt='spinner' />
-  </div>
-);
+export function Spinner() {
+  return <div style={{textAlign: 'center', height: '44px', width: '180px'}}>
+    <img src={loadingIndicator} alt='spinner'/>
+  </div>;
+}
```

Comment:
This is a slightly different version of https://github.com/DataBiosphere/duos-ui/pull/2512/files#diff-2b35d22b68311c48b29ae44312e7ccb0450ef084cac25b5c9d72ed2f4ad2c8fd 
I found that using the full `(): JSX.Element` syntax was showing an ESLint error that didn't seem to make any sense.